std/os/bits/darwin.zig

```diff
@@ -1090,3 +1090,25 @@ pub const stack_t = extern struct {
     ss_size: isize,
     ss_flags: i32,
 };
+
+pub const RTLD_LAZY = 0x1;
```

Comment:
Shouldn't these constants be in `std/c/darwin.zig`?

Reply:
`std/c/darwin.zig` doesn't have any other constants. they're all in `std/os/bits/darwin.zig`.